test/e2e/interactive_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build interactive
```

Comment:
we need to remove this since obs API uses go 1.16

Reply:
I keep making this mistake.  Thank you for your patience.  My editor uses `gofmt` on save.  My new workflow is to hand-edit with `vi` before pushing, but I forgot.  I have fixed this in the latest push.

Note that the official guidance from Go in the [1.17 release notes](https://go.dev/doc/go1.17) says "... all Go files should be updated to have both forms with the same meaning."  Consider allowing both forms in Observatorium code, as long as both have the same meaning.

Reply:
I think we can also update obs/api to go 1.17 - as far as I understand there should be no problem with that. I opened an issue here: https://github.com/observatorium/api/issues/262 
Regarding your flow/editor, it may be possible to configure it to use go v1.16 per project, but something to look into